lib/widgets/timetable/exam_schedule_tile.dart

```diff
@@ -100,26 +99,25 @@ class ExamTile extends StatelessWidget {
                       const SizedBox(
                         height: 3.0,
                       ),
-                      // if (course.venue != null)
-                      //   if(course.venue!.isNotEmpty)
-                      //     Row(
-                      //       children: [
-                      //         const Icon(
-                      //           FluentIcons.location_12_filled,
-                      //           color: lBlue,
-                      //           size: 13,
-                      //         ),
-                      //         const SizedBox(
-                      //           width: 4,
-                      //         ),
-                      //         Expanded(
-                      //           child: Text(
-                      //             course.venue!,
-                      //             style: MyFonts.w400.size(13).setColor(lBlue),
-                      //           ),
-                      //         )
-                      //       ],
-                      //     )
+                      if (course.venue != null && course.venue!.isNotEmpty)
```

Comment:
It should be midsem or endsem venue appropriately

Reply:
fixed